WPMedia/fusion-news-theme-blocks
# TMEDIA 597 large promo headline styles mobile

## Description
Match mocks with font-size and lineheight with design

## Jira Ticket
- [TMEDIA-597](https://arcpublishing.atlassian.net/browse/TMEDIA-597)

## Acceptance Criteria
The large promo headlines has incorrect font-size and line-height CSS values for mobile breakpoints

As per the designs (see screenshot) the values should be:

Font Size - 24px

Line Height - 32px

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-597-large-headline-promo-line-fix`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/shared-styles`
3. Go to http://localhost/pf/test-all-promos/?_website=the-gazette 
4. Decrease the viewport to mobile 
5. INspect the large promo headline to see what calculateRem sets. Note this is set using calculateRem but inspect shows the px output

## Effect Of Changes
### Before
<img width="1315" alt="Screen Shot 2021-12-20 at 14 28 35" src="https://user-images.githubusercontent.com/5950956/146828952-2d3b04e0-0165-4617-a2c9-a9328cfb4b12.png">
<img width="670" alt="Screen Shot 2021-12-20 at 14 28 27" src="https://user-images.githubusercontent.com/5950956/146828954-87613d06-f16d-4190-a89f-48cc0deb0018.png">

### After
<img width="691" alt="Screen Shot 2021-12-20 at 14 32 25" src="https://user-images.githubusercontent.com/5950956/146829128-e7c19d52-ecaf-4a01-93ca-8352521732b8.png">


<img width="708" alt="Screen Shot 2021-12-20 at 14 32 35" src="https://user-images.githubusercontent.com/5950956/146829125-2cce394f-c9ee-4561-a401-0f6a9bfd7004.png">


## Dependencies or Side Effects
- none

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
  - [ ] Confirmed this PR has unit test files -> na
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
